### PR TITLE
MRG fix for CNTEventType1 that don't have Latency

### LINKED ISSUE #7248: 'CNTEventType1' object has no attribute 'Latency'
Hi, 

I am trying to read raw EEG from .cnt files. However, Python3 returns the below error. The dataset I am reading is from UC San Diego https://sccn.ucsd.edu/~arno/fam2data/publicly_available_EEG_data.html. 

The dataset can be easily read by EEGLAB, however, I try to do everythig in Python3. 
If comment the lines shown in the Errors, I still receive the correct result. 

`Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/mne/io/cnt/cnt.py", line 169, in read_raw_cnt
    preload=preload, stim_channel=stim_channel, verbose=verbose)
  File "/usr/local/lib/python3.7/site-packages/mne/io/cnt/cnt.py", line 429, in __init__
    _read_annotations_cnt(input_fname, data_format=data_format))
  File "/usr/local/lib/python3.7/site-packages/mne/io/cnt/cnt.py", line 88, in _read_annotations_cnt
    duration = np.array([e.Latency for e in my_events], dtype=float)
  File "/usr/local/lib/python3.7/site-packages/mne/io/cnt/cnt.py", line 88, in <listcomp>
    duration = np.array([e.Latency for e in my_events], dtype=float)
AttributeError: 'CNTEventType1' object has no attribute 'Latency'`

### PR DESCRIPTION
**Pull request by agramfort**
closes #7248

@quocdat32461997 can you have a look and test? I tried comparing the data with EEGLAB but I get here 101 events while EEGLAB gives me 100 (it skips the last one).

### CODE REVIEW

File: mne/io/cnt/cnt.py [hunk 1]
```diff
@@ -83,7 +83,11 @@ def _translating_function(offset, n_channels, event_type,
                                       n_channels=n_channels,
                                       event_type=type(my_events[0]),
                                       data_format=data_format)
-        duration = np.array([e.Latency for e in my_events], dtype=float)
```
Comment (larsoner) [inline mne/io/cnt/cnt.py]: I would have just done `getattr(e, 'Latency', 0) for e in my_events`

**Comment by agramfort** [comment]
@quocdat32461997 this fixes the pb with your file.

@larsoner feel free to MRG

**Comment by quocdat32461997** [comment]
Hi MNE team, 
The codes work for now. Hopefully, you guys update MNE on pip3 soon.
@agramfort I'll check the events.

**Comment by agramfort** [comment]
in the mean time you can use:

pip install -U https://api.github.com/repos/mne-tools/mne-python/zipball/master

**Pull request merged**